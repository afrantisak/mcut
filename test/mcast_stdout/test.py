import os, sys, subprocess, time

proc = subprocess.Popen(["bin/recorder", "0.0.0.0", "224.1.1.1", "5000"])

import socket
address = ('224.1.1.1', 5000)
sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP)
sock.setsockopt(socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, 2)
time.sleep(0.2)
sock.sendto("1234567890", address)
time.sleep(0.2)
sock.close()
proc.kill()