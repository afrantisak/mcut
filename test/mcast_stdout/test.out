Listening on 0.0.0.0 224.1.1.1:5000
1234567890
